@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -901,7 +907,12 @@ export class VisibleTileSet {
                     tile.tileKey,
                     tile.offset
                 );
-                if (tile.hasGeometry || defaultSearchDirection === SearchDirection.NONE) {
+                tile.isFallback = false;
+                if (
+                    (tile.hasGeometry || defaultSearchDirection === SearchDirection.NONE) &&
+                    (!this.excludeTiles.has(tile.tileKey.mortonCode()) ||
```

Comment:
this change is just a debug change, it is to test the fallback, basically, if you put a breakpoint here, and add a morton code to the excludeTiles, then you can test the fallback manually.